packages/components/src/text/mixins.js

```diff
@@ -137,7 +137,7 @@ const variant = ( variantName ) => {
 
 /**
  * @typedef {Object} TextProps
- * @property {TextVariant} variant
+ * @property {TextVariant} variant - one of TextVariant to be used
```

Comment:
WordPress documentation standards don't use the `-`

https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/javascript/

```suggestion
 * @property {TextVariant} variant One of TextVariant to be used
```

Reply:
Fixed.